cowbird/services/impl/catalog.py

```diff
@@ -62,6 +62,16 @@ def on_created(self, filename):
         :param filename: Relative filename of a new file
         """
         LOGGER.info("The following file [%s] has just been created", filename)
+        # TODO This is a first naive implementation. Needs to be revised and improved.
```

Comment:
This implementation should be in `impl/geoserver.py`
All you need to do is : 
-  Implement `FSMonitor` from `Geoserver` like here : https://github.com/Ouranosinc/cowbird/blob/master/cowbird/services/impl/catalog.py#L15
-  Register/unregister the `Geoserver` class like here : https://github.com/Ouranosinc/cowbird/blob/master/cowbird/services/impl/catalog.py#L38, https://github.com/Ouranosinc/cowbird/blob/master/cowbird/services/impl/catalog.py#L42
- Implement on_created, on_deleted and on_modified. Basically, moving the code from here.